src/ALE/MOM_regridding.F90

```diff
@@ -871,8 +871,12 @@ subroutine regridding_main( remapCS, CS, G, GV, h, tv, h_new, dzInterface, frac_
       call calc_h_new_by_dz(CS, G, GV, h, dzInterface, h_new)
 
     case ( REGRIDDING_RHO )
-      if (do_convective_adjustment) call convective_adjustment(G, GV, h, tv)
-      call build_rho_grid( G, GV, G%US, h, tv, dzInterface, remapCS, CS )
+       if (do_convective_adjustment) call convective_adjustment(G, GV, h, tv)
+       if (use_ice_shelf) then
```

Comment:
Because frac_shelf_h is an optional argument both coming in and for `build_rho_grid()`, this 5 line block could be written as just the single line `call build_rho_grid( G, GV, G%US, h, tv, dzInterface, remapCS, CS, frac_shelf_h )`.